pkg/kvstore/client.go

```diff
@@ -55,5 +61,25 @@ func initClient() error {
 		panic("BUG: kvstore backend not specified")
 	}
 
+	l, err := CreateLease(LeaseTTL)
+	if err != nil {
+		Client = nil
+		return fmt.Errorf("Unable to create lease: %s", err)
+	}
+	leaseInstance = l
+
 	return nil
 }
+
+// CloseClient closes the kvstore client
+func CloseClient() {
+	if Client != nil {
```

Comment:
This is not threadsafe, if called from another thread than the one that called initClient.